src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```diff
@@ -1147,5 +1240,19 @@ public void Dispose()
                 }
             }
         }
+
+        public bool SupportMissingAttachments
+        {
+            get
+            {
+                switch (_protocolVersion)
+                {
+                    case 33:
```

Comment:
If we will 'split' the versioning then we can assume that all `_protocolVersion >= 33 && _protocolVersion <= 40999` will support this?

Reply:
We are better off with checking only explicitly known version, not ranges